docs/api-documentation.md

```diff
@@ -849,34 +847,51 @@ Delete a `Database`.
 
 ## `GET /api/database/`
 
-Fetch all `Databases`. `include_tables` means we should hydrate the Tables belonging to each DB. `include_cards` here
-  means we should also include virtual Table entries for saved Questions, e.g. so we can easily use them as source
-  Tables in queries. Default for both is `false`.
+Fetch all `Databases`.
+
+  * `include=tables` means we should hydrate the Tables belonging to each DB. Default: `false`.
+
+  * `saved` means we should include the saved questions virtual database. Default: `false`.
+
+  * `include_tables` is a legacy alias for `include=tables`, but should be considered deprecated as of 0.35.0, and will
+    be removed in a future release.
+
+  * `include_cards` here means we should also include virtual Table entries for saved Questions, e.g. so we can easily
+    use them as source Tables in queries. This is a deprecated alias for `saved=true` + `include=tables` (for the saved
+    questions virtual DB). Prefer using `include` and `saved` instead. 
 
 ##### PARAMS:
 
 *  **`include_tables`** value may be nil, or if non-nil, value must be a valid boolean string ('true' or 'false').
 
 *  **`include_cards`** value may be nil, or if non-nil, value must be a valid boolean string ('true' or 'false').
 
+*  **`include`** null
```

Comment:
Is this all this needs to say?

Reply:
No, I think there might be something wrong with the generator:
https://github.com/metabase/metabase/blob/e1746b4f66f6dc77a4acca84be3f403b0cd48853/src/metabase/api/database.clj#L190
I guess it should generate something like this?
``` *  **`include`** value may be nil, or if non-nil, value must be `tables` ```
Fixing it manually will remove it next time we run it, so we need to fix the generator.
But I don't understand how the generator works - Clojure.

Reply:
```suggestion
*  **`include`** value may be nil, or if non-nil, value must be `tables`
```